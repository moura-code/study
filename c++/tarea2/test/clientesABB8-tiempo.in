alturaClientesABBTiempo 100000 1
obtenerExisteClienteClientesABBTiempo 200000 0.01
Fin
