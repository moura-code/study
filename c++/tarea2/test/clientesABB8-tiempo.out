Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> La altura del clientesABB es 17. Calculado correctamente en menos de 1s.
2> Se obtuvieron los clientes? 1 1 1 1 con ids respectivos 0 199999 66666 133333
Calculado correctamente en menos de 0.010s.
3> Fin.
